src/components/draft-js-editor/DraftJSEditor.js

```diff
@@ -79,6 +80,8 @@ class DraftJSEditor extends React.Component<Props> {
 
     labelID = uniqueId('label');
 
+    commentFormTipID = uniqueId('comment-form-tip');
```

Comment:
Can we make this naming and prefix more generic, such as `descriptionID`? This component is used elsewhere outside of the CommentForm.